custom-tests.json

```diff
@@ -1452,7 +1452,7 @@
       "__base": "<%api.EXT_disjoint_timer_query:ext%> var instance = ext.createQueryEXT();"
     },
     "WebSocket": {
-      "__base": "var instance = new WebSocket('wss://mdn-bcd-collector.appspot.com');"
+      "__base": "var instance = new WebSocket('wss://' + location.hostname);"
```

Comment:
@gsnedders I looked at `location.host` or `location.hostname` here, the difference being whether the port is included for cases like serving on http://localhost:8080. Since the scheme here is already hardcoded to wss:// my thinking is that including the port will sometimes be wrong. And by not included it there's a requirement to run this on port 443, effectively.

This doesn't matter until there's a WebSocket server to talk to, of course.